ci/build_docs.sh

```diff
@@ -1,33 +1,30 @@
 #!/bin/bash
-# Copyright (c) 2023, NVIDIA CORPORATION.
+# Copyright (c) 2023-2025, NVIDIA CORPORATION.
 
 set -euo pipefail
 
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+
 rapids-logger "Create test conda environment"
 . /opt/conda/etc/profile.d/conda.sh
 
-export UCXX_VERSION="$(head -1 ./VERSION)"
```

Comment:
https://github.com/rapidsai/ucxx/blob/67f8f48daddc6474316496cb10755ef8391be7eb/cpp/doxygen/Doxyfile#L41

I think this needs to be exported in this job, for Doxygen.